source/layouts/layout.slim

```diff
@@ -57,3 +57,6 @@ html
         .footer-up
           a.caret-up href="#"
             img.caret-up-img src="/assets/components/footer/icon-arrow-up.svg" /
+
+  javascript:
```

Comment:
please use window.onload event for JS libraries that are not required for page rendering